### PR TITLE
[3.0] Dialog components and Full-screen Dialog

### PR DESCRIPTION
**Pull request by SiravitPhokeed**
<p>
  
  
  
</p>

**Features**
- New components
  - Dialog
  - Dialog Header
  - Dialog Content
  - Full-screen Dialog

### CODE REVIEW

File: apps/demo/pages/containers.tsx [hunk 1]
```diff
@@ -11,116 +12,264 @@ import {
...
+      <Dialog open={showRemStudents} onClose={() => setShowRemStudents(false)}>
+        <DialogHeader
+          title="Remove students?"
+          desc="The following students will no longer have access to the
+            organization “Kornor.”"
+        />
+        <DialogContent>
+          <List divided>
+            <ListItem align="center" lines={1}>
+              <ListItemContent title="Sadudee Theparree" />
```
Comment (Jimmy-Tempest) [inline apps/demo/pages/containers.tsx:120]: 😳

**Review by Jimmy-Tempest — APPROVED** [review]
LGTM!!!

**Comment by SiravitPhokeed** [comment]
I’m afraid this PR is not open for review yet. That’s fine though, I’m fixing minor accessibility issues right now and you can just review that part when I’m done.

File: apps/demo/pages/containers.tsx (hunk 1 above)
Comment (SiravitPhokeed) [reply apps/demo/pages/containers.tsx:120]: this was literally in the Specs
did you not see it

Comment (Jimmy-Tempest) [reply apps/demo/pages/containers.tsx:120]: Yes

**Review by Jimmy-Tempest — APPROVED** [review]
LGTM!!

**Pull request merged**